train_model.py

```diff
@@ -439,10 +476,10 @@ def get_params(self) -> dict:
     total_loss = 0
     total_samples = 0
     with torch.inference_mode():
-        for inputs, targets in tqdm(test_loader, desc="Test"):
+        for batch_seqs in tqdm(test_loader, desc="Test"):
```

Comment:
rename this test_batch_seqs to match val_batch_seqs, as seen above for consistency.